todoman/ui.py

```diff
@@ -373,16 +373,16 @@ def compact(self, todo):
         return json.dumps(data, sort_keys=True)
 
     def parse_priority(self, priority):
```

Comment:
Great! Please also add some tests for this method (there's `tests_porcelain` in a separate file).